redfish-core/lib/bios.hpp

```diff
@@ -445,7 +448,16 @@ inline void requestRoutesBiosSettings(App& app)
                             return;
                         }
 
-                        std::string biosAttrType = (*it).second;
+                        bool biosAttrReadOnlyStatus = ((*it).second).first;
+                        if (biosAttrReadOnlyStatus == true)
+                        {
+                            BMCWEB_LOG_ERROR
+                                << "Attribute Type is ReadOnly. Patch failed!";
+                            messages::internalError(asyncResp->res);
```

Comment:
I don't think this should be internal Error, how about PropertyNotWritable ?
https://redfish.dmtf.org/registries/Base.1.10.0.json

Reply:
done